### PR TITLE
Don't collapse columns with large-offset (fix #307)

### LINKED ISSUE #307: [The Grid] Column offsets are over-ridden when using collapse
Hi,

If using the `collapse` class on a row, if you try to offset a column within that particular row, the offsets' `padding-left` left value is over-ridden by the `collapse` padding.

### PR DESCRIPTION
**Pull request by jsit**

### CODE REVIEW

File: scss/grid/_grid.scss [hunk 1]
```diff
@@ -116,12 +116,14 @@ th.column {
   }
 
   //Collapsed logic
-  .collapse {
-    > tbody > tr > td.large-#{$i},
-    > tbody > tr > th.large-#{$i} {
-      padding-right: 0;
-      padding-left: 0;
-      width: -zf-grid-calc-px($i, $grid-column-count, $global-width) + $global-gutter;
+  .collapse > tbody > tr {
```
Comment (DanielRuf) [inline scss/grid/_grid.scss:119]: Why the change here?

Comment (jsit) [reply scss/grid/_grid.scss:119]: To reduce repetition; instead of:

```scss
 .collapse {
     > tbody > tr > td.large-#{$i},
     > tbody > tr > th.large-#{$i} {
```

the selector can just be:

```scss
 .collapse > tbody > tr {
     > td.large-#{$i},
     > th.large-#{$i} {
```

This way we're only typing the `tbody > tr` once.
